engine/apps/api/serializers/organization.py

```diff
@@ -122,44 +91,9 @@ def get_env_status(self, obj):
         phone_provider_config = get_phone_provider().flags
         return {
             "telegram_configured": telegram_configured,
-            "twilio_configured": phone_provider_config.configured,  # keep for backward compatibility
```

Comment:
not referenced anywhere ([search](https://github.com/search?q=repo%3Agrafana%2Foncall+twilio_configured&type=code))